.editorconfig

```diff
@@ -9,7 +9,7 @@ trim_trailing_whitespace = true
 insert_final_newline = true
 
 [*.py]
-profile = black # For isort
+profile = black
```

Comment:
Why this?

Reply:
I explained it in the PR description.

Reply:
Ugh... please then put the comment in the line above.

I prefer to put this comment here because it's not a standard editorconfig setting; it's just isort that reuses this file, so this comment should avoid confusions.

Reply:
Ok, done :+1: